data/strings/types_strings.txt

```diff
@@ -12416,37 +12416,84 @@
     uk = Перевал
 
   [type.natural]
-    en = Natural
+    en = Nature
```

Comment:
- It could actually, if our *.mapcss files have dedicated drawing rule (symbol, line or area filling) for [natural] type, but they don't.
- Yes, I suppose so.